src/pages/_app.js

```diff
@@ -21,6 +22,7 @@ export default function MyApp({
       <Head>
         <meta name="viewport" content="initial-scale=1, width=device-width" />
       </Head>
+      <GoogleAnalytics />
```

Comment:
This and it's import can be removed since it will be need to be added into the document's head.

Additionally, the code written thus far only initializes the analytics - but does not actually dispatch any page view tracking events.

We will need each page to dispatch an analytics call once on load.